telemetry-all/build.gradle.kts

```diff
@@ -0,0 +1,50 @@
+private object Versions {
+        const val slf4j = "1.7.30"
+        const val gson = "2.8.6"
+}
+
+plugins {
+    java
+    id("java-library")
+}
+
+configure<JavaPluginConvention> {
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
+}
+
+sourceSets {
+    main {
+        java.setSrcDirs(listOf(
+                "src/main/java",
+                "../telemetry_core/src/main/java",
+                "../telemetry/src/main/java",
+                "../telemetry-http-java11/src/main/java"
+        ))
+        resources.setSrcDirs(listOf(
+                "../telemetry_core/src/main/resources",
+                "../telemetry/src/main/resources",
+                "../telemetry-http-java11/src/main/resources"
+        ))
+    }
+}
+dependencies {
+    api("org.slf4j:slf4j-api:${Versions.slf4j}")
+    implementation("com.google.code.gson:gson:${Versions.gson}")
+}
+
+//plugins.withType<JavaPlugin>().configureEach {
```

Comment:
let's not put in commented-out code, if we can help it. If this is important to leave here, it needs a comment to describe why, and what it would take to get it uncommented (and an issue created to track that work).